docs/docker/redirects.conf

```diff
@@ -64,6 +64,7 @@ rewrite ^/((\d+\.\d+|next)/)?tutorial-performance-tips/?$ /docs/$1performance/ p
 rewrite ^/((\d+\.\d+|next)/)?tutorial-release-notes/?$ /docs/$1release-notes/ permanent;
 rewrite ^/((\d+\.\d+|next)/)?tutorial-migration-guide/?$ /docs/$1migration-from-7.4-to-8.0/ permanent;
 rewrite ^/((\d+\.\d+|next)/)?tutorial-known-limitations/?$ /docs/$1third-party-licenses/ permanent;
+rewrite ^/((\d+\.\d+|next)/)?hello-world/?$ /docs/demo/ permanent;
```

Comment:
Right! Otherwise, it would always redirect to the latest version. Fixed: [fc09db1](https://github.com/handsontable/handsontable/pull/8982/commits/fc09db127611be4fa43ecc0d0d375ebec22df0cd)